code/js/modules/BaseController.js

```diff
@@ -219,43 +356,48 @@
         (this.selectors.pause && this.doc().querySelector(this.selectors.pause))
       ),
       canPlayNext: this.overridePlayNext || !!(this.selectors.playNext && this.doc().querySelector(this.selectors.playNext)),
+      canMute: this.canMute,
       canLike: !!(this.selectors.like && this.doc().querySelector(this.selectors.like)),
+      canSeek: this.canSeek,
+      canSetVolume: this.canSetVolume,
       hidePlayer: this.hidePlayer
     };
+    state.volume = this.getVolume();
```

Comment:
```suggestion
      hidePlayer: this.hidePlayer,
      volume: this.getVolume()
    };
```
Any reason this is happening outside of the brace?